### ISSUE TITLE
Is it possible to have a  event like onFollow 、 onLoyalpoint ?

### ISSUE THREAD

**Issue by TIKER504** [instafluff/ComfyJS#16]
First, this is an awesome repositories that can be applied easily,

but i think it will be perfect if have a event like "onFollow" or "onLoyalpoint",

thank you.

**Comment by Zer0Lexaros** [comment]
I would like to second this request as it would be handy to be able to trigger an event on follow

**Comment by TIKER504** [comment]
> I would like to second this request as it would be handy to be able to trigger an event on follow

HA HA~~ thank you so much :D

**Comment by MichaelJolley** [comment]
I don't think this is possible in the current iteration.  ComfyJS relies on tmi.js, which is connecting to Twitch chat to receive all events.  Currently, follows aren't emitted in Twitch chat which would mean ComfyJS would have to get that information another way (i.e. webhooks/pubsub)

**Comment by instafluff** [comment]
Yup! It's not possible to get those notifications without PubSub at the moment (which requires OAuth) but it would be great to get PubSub support working in ComfyJS

**Comment by instafluff** [comment]
Did a bit more investigating and it looks like follow events aren't available via PubSub either, only webhooks which require a backend server... https://discuss.dev.twitch.tv/t/api-request-add-new-follower-topic-to-pubsub/26852/2

Could be worth all of us voting for this feature. I'm surprised the webhook topics aren't PubSub topics (follower, title and game changes, extension actions, etc.)
